src/main/java/com/checkscanner/check/CheckApplication.java

```diff
@@ -4,16 +4,13 @@
 import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.ComponentScan;
-import java.util.Map;
 
 @ComponentScan(basePackages = "com.checkscanner.check")
 @EnableAutoConfiguration
 @SpringBootApplication
-public class CheckApplication
-{
+public class CheckApplication {
 
-	public static void main(final String[] args)
-	{
+	public static void main(final String[] args) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AUndocumentedApi)